webserver/webserver.go

```
@@ -30,44 +42,47 @@ func (ws *WebServer) Start() {
-func (ws *WebServer) handleRequest(conn net.Conn) *Response {
+func (ws *WebServer) handleRequest(conn net.Conn) {
+	reader := bufio.NewReader(conn)
 	// Read the request line
 	// GET /path HTTP/1.1
-	reader := bufio.NewReader(conn)
 	requestLine, err := reader.ReadString('\n')
 	if err != nil {
 		log.Println("Error reading request line:", err)
-		return NewResponse(http.StatusBadRequest, []byte("Bad Request"))
+		response := NewResponse(http.StatusBadRequest, []byte("Bad Request"))
+		response.WriteTo(conn)
```

golangci-lint (1.62.2)

Error return value of `response.WriteTo` is not checked

(errcheck)